website/src/content/blog/2025-03-31-typespec-1-0-release/typespec_1_0_release.md

```diff
@@ -0,0 +1,215 @@
+public virtual async Task<IActionResult> GetTodoItems()
+{
+    var result = await TodoItemsOperationsImpl.GetTodoItemsAsync();
+    return Ok(result);
+}
+```
+
+### Client libraries
+
+TypeSpec can also generate client libraries in multiple languages, providing a consistent, type-safe interface to your API:
+
+```javascript
+const client = new TodoServiceClient();
+const todo = await client.createTodoItem({
+  content: "Buy groceries",
+  dueDate: new Date("2025-03-31"),
+  isCompleted: false,
+});
+console.log(todo);
+```
+
+### Current limitations
+
+The code generation features are currently in preview and have a few technical limitations to be aware of:
+
+- Limited support for authentication mechanisms
+- Complex data transformations may require manual coding
+- Not all HTTP features are fully supported
+- Documentation for these features is still being developed
+
+We're actively working to address these limitations and we welcome your feedback.
+
+## Emitters: From TypeSpec to different formats
+
+TypeSpec's emitter system allows your API definitions to be transformed into various formats. The 1.0-RC includes stable emitters for OpenAPI 3.0 and JSON Schema, with additional emitters in preview.
+
+The OpenAPI 3.0 emitter is particularly important as it ensures compatibility with existing OpenAPI workflows and toolchains, allowing teams to adopt TypeSpec while maintaining their current API ecosystem. This means you can gradually introduce TypeSpec into your development process without disrupting existing tools and processes that depend on OpenAPI specifications.
+
+### Custom emitters with the emitter framework
+
+For teams with specialized needs, TypeSpec 1.0-RC includes a new experimental emitter framework that allows you to build custom emitters. This framework provides access to TypeSpec's compiler APIs and type system, enabling you to transform your TypeSpec definitions into any format your organization needs.
+
+The emitter framework is designed to be extensible and developer-friendly, with clear patterns for accessing the TypeSpec type system, transforming types, and generating output. This opens up possibilities for creating custom emitters for internal specifications, proprietary formats, or integration with existing toolchains.
+
+For teams with existing OpenAPI definitions, we've created an [OpenAPI to TypeSpec converter tool](https://typespec.io/docs/emitters/openapi3/cli/) to help migrate existing API definitions to TypeSpec.
+
+## Community support and contributions
+
+TypeSpec is an [open source project](https://github.com/microsoft/typespec) developed by Microsoft and shared with the community. We're working to build a vibrant ecosystem around TypeSpec with multiple channels for support, collaboration, and knowledge sharing:
+
+- **[GitHub Issues](https://github.com/microsoft/typespec/issues)**: For bug reports, feature requests, and technical discussions
```

Comment:
I'd recommend a separate line for GH issues and then GH discussions (which is where the community page sends people)